### ISSUE TITLE
"Ignore streaming service setting recommendations" checkmark does not seem to work

### ISSUE THREAD

**Issue by Aktanusa** [obsproject/obs-studio#4125]
## Platform
Operating system and version: Windows 10 2004
OBS Studio version: 26.1.1

## Expected Behavior
Checking "Ignore streaming service setting recommendations" should work.  It doesn't since I choose a rate control of CRF for Twitch and it is still set to CBR.  I have noticed that the setting "ApplyServiceSettings=false" is missing from the [AdvOut] section of basic.ini.  After manually adding "ApplyServiceSettings=false", the rate control is finally set to CFR.

## Current Behavior
Checking "Ignore streaming service setting recommendations" adds "IgnoreRecommended=true" in [Stream1] section but does not add "ApplyServiceSettings=false" in the [AdvOut] section of basic.ini.  I'm not sure if is suppose to be there anyway since maybe it's a bug that the new flag "IgnoreRecommended=true" does not work.

## Steps to Reproduce
1. Install OBS
2. Start OBS
3. Skip wizard
4. Click on Settings
5. Log into Twitch in Stream section
6. Check "Ignore streaming service setting recommendations"
7. Goto Output section and change Output Mode to Advanced
8. Change Encoder to x264, change Rate Control to CRF, make sure CRF number is 23, change Keyframe Interval to 2.
9. Click Okay and start streaming
10. Goto View -> Stats and notice the bitrate is 6000 kb/s instead of under 100 kb/s since you are streaming a black screen.
11. Check logs and see "rate_control: CBR" under "[x264 encoder: 'streaming_h264'] settings:"

**Comment by dodgepong** [comment]
You have to enable Advanced output mode to fully disable streaming service recommendations. That checkbox only ignores a portion of settings, such as bitrate and resolution. Twitch don't want you streaming with CRF and expect your streams to be encoded a particular way so we do what we can to prevent you from violating what services are expecting. We have no plans to make it easier for people to disable these recommendations.

This issue should be treated as a request to clarify wording, not change behavior.

**Comment by dodgepong** [comment]
Sorry, I just checked and didn't realize that we changed the UI in Advanced mode in 26.1. I'm not sure why the checkbox in Advanced settings was removed.

**Comment by Aktanusa** [comment]
I think I read somewhere that the setting in the advanced area has been moved to the stream section and renamed to "Ignore streaming service setting recommendations".  Basically it's the same flag, but the work for the move was not completely done and has bugs.  Just an educated guess.

**Comment by naanlizard** [comment]
Can confirm we're seeing this bug too. Bad change to move the checkbox IMO

**Comment by kkartaltepe** [comment]
Loss of the UI breaks override for users of linux where many newer intel chips dont support CBR. So even a well chosen CQP stream will just fail to initialize

**Comment by wllyng** [comment]
Hi I would like to work on this issue if it is still open! What is a good place to start?

**Comment by kkartaltepe** [comment]
Yes this is still an issue, you could bisect this to see what commit introduced this new behavior. From there you could work on re-introducing the functionality that was lost. As mentioned this happened somewhere around 26.0 -> 26.1
